packages/fast-form-generator-react/src/form/form-item.align-horizontal.tsx

```diff
@@ -21,7 +21,7 @@ class FormItemAlignHorizontal extends React.Component<IFormItemComponentMappingT
                 >
                     {this.props.label}
                 </label>
-                <div>
+                <div className={this.props.managedClasses.formItemAlignHorizontal_input}>
```

Comment:
@nicholasrice technically this isn't the input, should we still be using this type of naming?

Reply:
yea it should probably be something more like "inputContainer"